### DISCUSSION TITLE
Failed to auto-install CurseForge modpack

### DISCUSSION THREAD

**Discussion by RavMarv** [itzg/docker-minecraft-server#1940] [Q&A]
I'm trying to make a minecraft server running 'adventure pack' (a fabric modpack from curseforge), and every time I try to open it, it crashes after trying to install the mod.

LOG:
(many successful downloads before)
[mc-image-helper] 22:01:02.249 INFO  : Downloaded mod file mods/mobz-2.4.6.jar
[mc-image-helper] 22:01:02.322 INFO  : Downloaded mod file mods/Mambience-3.0.0+1.16.5.jar
[mc-image-helper] 22:01:02.788 INFO  : Downloaded mod file mods/better-end-0.9.6-pre.jar
[mc-image-helper] 22:01:05.489 INFO  : Installing Fabric Versions(game=1.16.5, loader=0.11.6, installer=0.11.1)
[mc-image-helper] 22:01:05.587 ERROR : 'install-curseforge' command failed. Version is 1.24.20
me.itzg.helpers.http.FailedRequestException: HTTP request of https://meta.fabricmc.net/v2/versions/loader/1.16.5/0.11.6/0.11.1/server/jar failed with 400 Bad Request: Extracting filename

I looked into the link and it couldn't find the file (it seems like '0.11.1' doesn't exist). And suggestions?

EDIT: this is the version and type im using if its important:
"-e TYPE=AUTO_CURSEFORGE -e CF_PAGE_URL=https://www.curseforge.com/minecraft/modpacks/fabric-adventure-pack"

**Comment by itzg** [comment]
Thanks for pointing this out. I created https://github.com/itzg/docker-minecraft-server/issues/1941 to track the bug that is happening here.

**Reply by itzg** [reply]
When this build finishes, the published image should properly handle the Fabric version referenced by the modpack

https://github.com/itzg/docker-minecraft-server/actions/runs/4080715318
